### ISSUE TITLE
Améliorer le mail d'invitation que reçoivent les agents zéro lors de l'onboarding

### ISSUE THREAD

**Issue by mekaidmekaid** [betagouv/rdv-solidarites.fr#3559]
### Définition du problème ❓ 

Le template du mail d'invitation en place aujourd'hui est obsolète et pas en phase avec les ambitions du projet en cours.

![Image](https://github.com/betagouv/rdv-solidarites.fr/assets/122271472/470b5450-40b3-4643-88cc-6c7c4cb6076f)

### Solution 💡 

Remplacer ce template par celui-ci avec l'identité RDV Commune : 

https://www.figma.com/file/0er4UN7N3R37Ag34Aq1GTX/Refonte?type=design&node-id=124%3A32463&t=xrexNY3GfRXcgPiP-1

**Comment by mekaidmekaid** [comment]
@Teodora-Stanki ce mail doit être revu à la fois au niveau du contenu et du design, je le met en rediscuss qu'en puisse en parler ensemble

**Comment by Teodora-Stanki** [comment]
@mekaidmekaid Yes c'est prevu, le template a déjà été refondu : https://www.figma.com/file/0er4UN7N3R37Ag34Aq1GTX/Refonte?type=design&node-id=124%3A32463&t=xrexNY3GfRXcgPiP-1

**Comment by mekaidmekaid** [comment]
Arf, je n'ai pas réalisé qu'il s'agissait de celui-ci, j'adapte le ticket ! 

Mercii

**Comment by mekaidmekaid** [comment]
Bonjour `Prénom`

Vous avez été invité à rejoindre RDV Service Public en tant qu'agent du service `Mairie`.

[Accepter l'invitation](https://r.email.rdv-solidarites.fr/tr/cl/_o2QZsEhCGBz-A-TWN6nXCkvksAxUHSGd8tWgsSWeGUW52gqqXEi7C5m8_OMTzhAunJSuRBtIc0XPdEqz-hmCDNpexNCSs18tJAdg7gWBjBfJCDNKq4LDXVSIJR09QR98bW-IBOux_UuuGglCzuak4l_wG0ZHq66LIxluh4WWoGDWOBvLNmiUEaqobdDuQi1T8aCILSMm3w7TZx1KcfuEFOgb1k9rvjycjS6Le216twfGD-Ui0c0K4ZnA0iu0wO3hr5iCfuXggqVwMufwhbBPSV6UQj7Cbd9_L3PqLscBC75nJmYY1df5CWeUGSrI2NDyoofD2lqUn_iPc-rA_5jQHJjy_4FGGADQ0HE31mVXb60FNBtMQwoyRnTuin_LwbkkGX-oeP7UNEbC8DzkOnAQZaq98fTeBRlzM5uPGHf904feWr9To5KzED2Hcvk)

Cette invitation est valable jusqu'au 04 juillet. Si vous ratez cette date mais que vous souhaiteriez accepter, veuillez demander à l'[équipe RDV Service Public (victor@ahaoho.io)](mailto:victor@ahaoho.io) de vous renvoyer l'invitation.

Si vous n'acceptez pas cette invitation, ignorez cet email. Aucun compte ne sera créé sans votre autorisation.

**Comment by victormours** [comment]
j'arrive pas à obtenir le bon mail en prod. Je finis une autre PR et je re-regarde ça

**Comment by mekaidmekaid** [comment]
@victormours je viens d'updater le message sur mon commentaire précédent ✅
